custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -320,7 +349,7 @@ def __init__(self, condition, values):
 
     def validate(self, dataframe):
 
-        is_sparse = dataframe.dtypes.apply(pd.api.types.is_sparse).any()
```

Comment:
so why was this a bad technique?

Reply:
it catches if there is a sparse series in the dataframe, not if the dataframe itself is sparse

Reply:
what does it mean for the dataframe to be sparse, if not that it has sparse series?

Reply:
like a csr dataframe object, which is what is actually received

Reply:
do you mean a csr matrix?

Reply:
yes